src/html.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+
+export default function HTML(props) {
+  return (
+    <html {...props.htmlAttributes}>
+      <head>
+        <meta charSet="utf-8" />
+        <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        {props.headComponents}
+      </head>
+      <body {...props.bodyAttributes} className="docs-page">
+        {props.preBodyComponents}
+        <noscript key="noscript" id="gatsby-noscript">
+          This app works best with JavaScript enabled.
+        </noscript>
+        <div
+          key={`body`}
+          id="___gatsby"
+          dangerouslySetInnerHTML={{ __html: props.body }}
+        />
+        {props.postBodyComponents}
+        <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
```

Comment:
Is this how Algolia recommends you load it? Can we add a script resource integrity nonce here? It's a pretty random domain, and I want to avoid loading a script that might've been tampered with. That's assuming we can rely on this particular script source never changing (strong versioning)

Reply:
@unclecheese Is this captured as a TODO somewhere?